hydrus/data/crud.py

```diff
@@ -563,6 +574,37 @@ def get_collection(API_NAME: str,
             object_template = {
                 "@id": "/{}/{}Collection/{}".format(API_NAME, type_, instance_.id), "@type": type_}
         collection_template["members"].append(object_template)
+
+    # If pagination is disabled then stop and return the collection template
+    if not paginate:

```

Comment:
all checks should be explicit by using `is`: i.e. `if var is None` or `if var is False` etc. this improves understandability.